test/roby_app/test_plugin.rb

```diff
@@ -218,20 +218,20 @@ def perform_app_assertion(result)
                     @server_threads = []
                     @process_servers = []
                     register_process_server("test_ps")
-                    Plugin.log_transfer_ip = "127.0.0.1"
-                    Plugin.setup_local_log_transfer_server(Roby.app)
+                    app.log_transfer_ip = "127.0.0.1"
+                    app.setup_local_log_transfer_server
                 end
 
                 after do
                     close_process_servers
-                    Plugin.log_transfer_ip = nil
+                    app.log_transfer_ip = nil
                 end
 
                 it "uploads file from Process Server" do
                     @ps_logfile = create_test_file(@ps_log_dir)
```

Comment:
Do  you need to store that in an instance variable ? I don't see it used anywhere.

Reply:
Do you mean having log_transfer_ip as an instance variable or specifically setting it to nil at that point?

Reply:
I think I get what you mean now. Do you think it should go back to Plugin?

Reply:
I meant @ps_logfile. Comments are applied to the last line of the code block.

Reply:
Oh okay